src/V2/Actor/Template/Tokenizer.php

```diff
@@ -20,6 +20,10 @@ class Tokenizer implements TokenizerInterface
         '\RelativeNamespace' => self::TOKEN_NAMESPACE_RELATIVE,
         'NamespacedPrimaryActorName' => self::TOKEN_PRIMARY_ACTOR_NAME_FULL_PASCAL,
         'PrimaryActorName' => self::TOKEN_PRIMARY_ACTOR_NAME_SHORT_PASCAL,
+        'NamespacedParentActorName' => self::TOKEN_PARENT_ACTOR_NAME_FULL_PASCAL,
+        'RelativeParentActorName' => self::TOKEN_PARENT_ACTOR_NAME_RELATIVE_PASCAL,
+        'ParentActorName' => self::TOKEN_PARENT_ACTOR_NAME_SHORT_PASCAL,
```

Comment:
or maybe at least leave a comment in the code here as to the ordering being important and what the reasoning is

Reply:
Yeah, I think any time there's a bunch of search/replace things happening, order should always be taken into account. There's just too many edge cases to deal with otherwise. I'll add comments so they aren't as likely to get moved, but that's also why we have tests.

An alternative (not sure how well it would work for the API) is to use something like `ShortPrimaryActorName` and `ShortParentActorName` to prevent substrings.